js/web/src/components/NavBar.js

```diff
@@ -197,11 +197,44 @@ const ConnectionDot = styled('span')(({ theme }) => ({
 }))
 
 const Logo = styled(NavLink)(() => ({
+  display: 'flex',
+  flexDirection: 'column',
+  justifyContent: 'center',
+  alignItems: 'center',
+  padding: '7px 15px',
   position: 'absolute',
-  left: '50%',
-  top: '0',
-  transform: 'translateX(-50%)',
-  height: '60px',
+  width: '76px',
+  height: '43px',
+  left: 'calc(50% - 76px/2)',
+  top: '0px',
+  '& span' : {
+    position: 'static',
+    width: '46px',
+    height: '13px',
+    left: '15px',
+    top: '15px',
+
+    fontFamily: 'Helvetica',
+    fontStyle: 'normal',
+    fontWeight: 'bold',
+    fontSize: '18px',
```

Comment:
these values will be inherited automatically from the theme by changing the span to an `<Typography variant="h3" >`